doc/release/1.16.0-notes.rst

```diff
@@ -14,6 +14,12 @@ New functions
 Deprecations
 ============
 
+`typeNA` and `sctypeNA` have been deprecated
+-------------------------------------------
+
+The type dictionaries `numpy.core.typeNA` and `numpy.core.sctypeNA` were buggy
+and not documented. They will be removed in the next release. Use 
```

Comment:
2 releases, not the next release

Reply:
the actual deprecation message is correct (1.18) so just needs correcting here

Reply:
fixed